css/components/app/pages/splash_detail.scss

```diff
@@ -2,73 +2,23 @@
   width: 100%;
   height: 100vh;
 
-  .header {
+  .hotspot-section {
     position: absolute;
+    transition: 1s;
     top: 0px;
     left: 0px;
     width: 100%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 26px;
-    z-index: 9999;
-
-    .logo {
-      width: 310px;
-    }
-
-    img, a {
-      cursor: pointer;
-    }
-
-    .links {
-      display: flex;
-      a {
-        color: white;
-        text-decoration: none;
-      }
-
-      .earth-view-link {
-        margin-right: 45px;
-
-        a {
-          margin-left: 15px;
-        }
-      }
-    }
-  }
-
-  .right-section {
-    position: absolute;
-    transition: 1s;
-    top: 0px;
-    left: 60%;
-    width: 40%;
     height: 100%;
-    background-color: rgba(0,0,0,0.49);
+    background-color: rgba(0,0,0,0.8);
     color: $white;
-    z-index: 9999;
-
-    .arrow-button {
-      position: absolute;
-      top: 0px;
-      left: -30px;
-      width: 30px;
-      height: 40px;
-      display: flex;
-      align-items: center;
-      justify-content: center;
-      background-color: $dark-pink;
-      border-radius: 4px 0 0 4px;
-      cursor: pointer;
-    }
+    z-index: 99999;
+    padding-left: 33%;
```

Comment:
These percentages... are a little bit weird... You only want to center it, don't you?

Reply:
Not really, this is just temporal since I'm waiting for the assets I'm currently missing https://projects.invisionapp.com/d/main#/console/12675639/265630875/preview